src/orquestra/integrations/braket/simulator/LocalSimulator.py

```diff
@@ -130,4 +118,8 @@ def get_exact_noisy_expectation_values(
                 rho = result_object.values[0]
                 expectation_value = np.real(np.trace(rho @ sparse_pauli_term_ndarray))
                 values.append(expectation_value)
+
         return expectation_values_to_real(ExpectationValues(np.asarray(values)))
+
+    def is_natively_supported(self, operation: Operation) -> bool:
+        return True
```

Comment:
I think that `BraketLocalSimulator` can only natively handle `GateOperatoin`s, right? Hence, this function should probably look somewhat like this:

```python
    def is_natively_supported(self, operation: Operation) -> bool:
        return isinstance(operation, GateOperaton)
```

Reply:
Done